lib/render/engineScript.js

```diff
@@ -56,9 +56,6 @@ module.exports = function (inputs, callback, done) {
     inputs.parentModuleDirectory
   ]
 
-  let consoleFromSandbox

```

Comment:
I'm guessing this was done in order to not trigger `no-use-before-define`, but it's actually accessed a bit up which makes that point moot... I'm open to other suggestions for this ☺️